response_operations_ui/views/messages.py

```diff
@@ -75,6 +75,7 @@ def view_conversation(thread_id):
     thread_conversation = message_controllers.get_conversation(thread_id)
     refined_thread = [_refine(message) for message in reversed(thread_conversation['messages'])]
     latest_message = refined_thread[-1]
+    my_conversations = request.args.get('my_conversations', default='false')
```

Comment:
```suggestion
    my_conversations = request.args.get('my_conversations', 'false')
```

I think this is the better way of doing it.  Less words is generally better and leaving out 'default=' is pretty standard.

If you agree, can you do this for all the other ones in here please?  Though after looking at the whole file, there's quite a bit of this so it might be worth leaving it.... your call ^^

Reply:
I kind of like the explicit default in this specific case. It tells the reader that if no param is found then it defaults to a specific value. It doesn't have a significant performance hit . When I looked I could see several calls made with named params out of order, i.e type given before default. So I modified calls to call default first , so if we do change all we have to do is remove 'default='